src/ao3/works.py

```diff
@@ -274,6 +344,9 @@ def json(self, *args, **kwargs):
         *args and **kwargs are passed directly to `json.dumps()` from the
         standard library.
 
+        bookmark_users is not provided 
+        due to time it takes to go through all the bookmarks
```

Comment:
Isn't it provided above?

Reply:
The first line of comment says that complete representation of work is provided when calling json(). But because of bookmark_users not being provided when calling json(), it is no longer a complete representation, which was the reason I wrote that line. 

Perhaps it should be reworded differently?